### DISCUSSION TITLE
DiD with variation in treatment timing

### DISCUSSION THREAD

**Discussion by vob2** [QuickPay-Operational-Performance/Data-and-code#64] [Ideas]
@vibhuti6 @harishk05 @JNing0 
I came across this discussion of DiD design: https://blogs.worldbank.org/impactevaluations/what-are-we-estimating-when-we-estimate-difference-differences 

which is based on this paper: https://cdn.vanderbilt.edu/vu-my/wp-content/uploads/sites/2318/2019/07/29170757/ddtiming_7_29_2019.pdf 

The picture looks exactly like our two-stage QuickPay implementation.  So I thought to share it with you.

**Comment by vibhuti6** [comment]
Hi Vlad, thank you for sharing these links. This paper was the main reference I had for why we can't use a standard fixed effects model for our full sample. The fix they suggest in their paper is to choose a control group which is "never treated" or to make sure that comparisons happen between "treated" and "not yet treated". In our setting, however, that is not possible because we have only two groups (small and large businesses) and both groups get treated. This is why I think there are issues with the second implementation of Quickpay and a "reverse difference in differences" may be more suitable.
